lib/stages/bufferCopy.hpp

```diff
@@ -0,0 +1,67 @@
+#ifndef BUFFER_COPY_HPP
+#define BUFFER_COPY_HPP
+
+#include "Config.hpp" // for Config
+#include "Stage.hpp"
+#include "buffer.h"
+#include "bufferContainer.hpp" // for bufferContainer
+#include "visUtil.hpp"
+
+#include <string> // for string
+#include <tuple>  // for tuple
+#include <vector> // for vector
+
+/**
+ * @brief Copies frames from one buffer into many buffers.
+ *
+ *
+ * @warning The sizes of the frames must be the same in all buffers, and the
+ *       metadata types and underlying pools must also be the same.
+ *
+ *
+ * @par Buffer
+ * @buffer in_buf Buffer to copy frames from
+ *        @buffer_format Matches the output buffers
+ *        @buffer_metadata Matches the output buffers
+ * @buffer out_bufs Array of output buffers to copy frames into.
```

Comment:
These comments appear to have been copied from `bufferMerge` which has an optional complex selection process.  This `stage` doesn't really need that fancy "internal_name" functionality, and including that in the comments suggests it might do something which it doesn't.  I would simplify this, and also simplify the config reading logic above.  We really just need the version without the internal names here.